auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -808,22 +808,38 @@ public T setLogo(@DrawableRes int logo) {
 
         /**
          * Specifies the terms-of-service URL for the application.
+         *
+         * @deprecated Please use {@link #setTosAndPrivacyPolicyUrls(String, String)}
+         * For the Tos link to be displayed a Privacy Policy url must also be provided.
          */
         @NonNull
+        @Deprecated
         public T setTosUrl(@Nullable String tosUrl) {
             mTosUrl = tosUrl;
             return (T) this;
         }
 
         /**
          * Specifies the privacy policy URL for the application.
+         *
+         * @deprecated Please use {@link #setTosAndPrivacyPolicyUrls(String, String)}
+         * For the Privacy Policy link to be displayed a Tos url must also be provided.
          */
         @NonNull
+        @Deprecated
         public T setPrivacyPolicyUrl(@Nullable String privacyPolicyUrl) {
             mPrivacyPolicyUrl = privacyPolicyUrl;
             return (T) this;
         }
 
+        @NonNull
+        public T setTosAndPrivacyPolicyUrls(@NonNull String tosUrl,
+                                            @NonNull String privacyPolicyUrl) {
```

Comment:
We should add a `Preconditions` check in here to check the non-null-ness since `@NonNull` is just an IDE hint.

Reply:
Done.